src/main/java/graphql/execution/Async.java

```diff
@@ -126,33 +127,6 @@
         return result;
     }
 
-    public static <T> void copyResults(CompletableFuture<T> source, CompletableFuture<T> target) {
-        source.whenComplete((o, throwable) -> {
-            if (throwable != null) {
-                target.completeExceptionally(throwable);
-                return;
-            }
-            target.complete(o);
-        });
-    }
-
-
-    public static <U, T> CompletableFuture<U> reduce(List<CompletableFuture<T>> values, U initialValue, BiFunction<U, T, U> aggregator) {
-        CompletableFuture<U> result = new CompletableFuture<>();
-        reduceImpl(values, 0, initialValue, aggregator, result);
-        return result;
-    }
-
-    public static <U, T> CompletableFuture<U> reduce(CompletableFuture<List<T>> values, U initialValue, BiFunction<U, T, U> aggregator) {
-        return values.thenApply(list -> {
-            U result = initialValue;
-            for (T value : list) {
-                result = aggregator.apply(result, value);
-            }
-            return result;
-        });
-    }
```

Comment:
@andimarek - do you want these methods?  They clearly have no tests or usage otherwise it would blow up